azure-pipelines.yml

```diff
@@ -95,7 +95,7 @@ jobs:
               pushd ${PROJECT} >/dev/null
               echo "Generating meta.json..."
               JSON_PATH="$(Agent.TempDirectory)/${NAME}.meta.json"
-              $(JULIA) ./build_tarballs.jl --meta-json="${JSON_PATH}"
+              $(JULIA) ./build_tarballs.jl --compile-min --meta-json="${JSON_PATH}"
```

Comment:
I _think_ we can safely use `--compile=min` here to speedup this operation?

Reply:
For example, for `Qt` I get 5.5 seconds with `--compile=min` and 18.4 seconds without it.

Reply:
Oooh, good idea.

Reply:
I think it's `--compile=min` though, not `--compile-min`.  ;)

Reply:
Yeah, I wrote it correct in the comments but not in the code :smirk: